docker-compose-mopidy.yml

```diff
@@ -5,13 +5,18 @@ services:
     container_name: mopidy
     build:
       context: ./mopidy
-    volumes:
-      - ./mopidy/config/mopidy.conf:/config/mopidy.conf
     command: mopidy
     ports:
       - "6600:6600"
       - "6680:6680"
     networks:
       default:
         aliases:
-          - mopidy.local
+          - mopidy
```

Comment:
I think it's still worth keeping to show intent, but this could be removed at this point because the container name is also `mopidy`.